### DISCUSSION TITLE
Prerelease 0.87.3-pre to release 0.87.3

### DISCUSSION THREAD

**Discussion by JustinParathazham** [Snowflake-Labs/terraform-provider-snowflake#2632] [General]
Hi all 

We are looking for a solution to an issue with grant ownership to database role and realize that the current pre release (https://github.com/Snowflake-Labs/terraform-provider-snowflake/releases/tag/v0.87.3-pre) contains a fix for this. Is there an approximate time frame, where this pre release gets to a proper release?

**Comment by sfc-gh-jcieslak** [comment]
Hey @JustinParathazham 👋 
Grant ownership most likely won't be added with 0.87.3-pre and even 0.87.3. If you look here: #2604, It was partially implemented. There's still much to cover as granting ownership comes with a lot of difficulties in implementation and it has a lot of edge cases to cover. I think It's safe to assume that the grant ownership resource should be available in ~2/3 weeks.
